src/App.jsx

```diff
@@ -1,5 +1,29 @@
+import FooterWrapper from './layouts/FooterWrapper/FooterWrapper';
+import HeaderWrapper from './layouts/HeaderWrapper/HeaderWrapper';
+import MiddleWrapper from './layouts/MiddleWrapper/MiddleWrapper';
+
+import 'normalize.css';
+import './App.css';
+import { BrowserRouter } from 'react-router-dom';
+import { useState } from 'react';
+import { THEME_DARK, THEME_LIGHT } from './constants/constants';
+
 function App() {
-  return <div>Hello world!</div>;
+  const [themeMode, setThemeMode] = useState(THEME_LIGHT);
+
+  function toggleThemeMode() {
+    setThemeMode(themeMode === THEME_LIGHT ? THEME_DARK : THEME_LIGHT);
+  }
+
+  return (
+    <BrowserRouter>
+      <div className={`global-wrapper theme-mode_${themeMode}`}>
```

Comment:
just a tip, you may also use something like 
```
className={['global-wrapper', `theme-mode_${themeMode}`].join(' ')}
```
There is also a library called classnames, but I think join thing covers everything you need.

it becomes more useful when you start applying classNames conditionally. Anyway try to stick to one approach